Makefile

```diff
@@ -85,7 +91,35 @@ run-docker:
 	docker run \
 		--net lms_default \
 		-e DATABASE_URL=postgresql://postgres@postgres/postgres \
-		-e FEATURE_FLAGS_COOKIE_SECRET \
+		-e GOOGLE_CLIENT_ID \
+		-e GOOGLE_DEVELOPER_KEY \
+		-e GOOGLE_APP_ID \
+		-e H_API_URL_PRIVATE \
+		-e H_API_URL_PUBLIC \
+		-e H_AUTHORITY \
+		-e H_CLIENT_ID \
+		-e H_CLIENT_SECRET  \
+		-e H_JWT_CLIENT_ID \
+		-e H_JWT_CLIENT_SECRET \
+		-e JWT_SECRET \
+		-e LMS_SECRET \
+		-e RPC_ALLOWED_ORIGINS \
+		-e VIA_URL \
+		-e SESSION_COOKIE_SECRET \
+		-e OAUTH2_STATE_SECRET \
+		-p 8001:8001 \
+		hypothesis/lms:$(DOCKER_TAG)
+.PHONY: run-docker-dev
+run-docker-dev:
```

Comment:
In order to use docker for dev with all other services running also in dev they all need to be on the same network (I chose to use h_default which is used in the h repo as the network for everything).